modules/oc-mirror-dry-run.adoc

```diff
@@ -8,63 +8,59 @@
 [id="oc-mirror-dry-run_{context}"]
 = Performing a dry run
 
-You can use oc-mirror to perform a dry run, without actually mirroring any images. This allows you to review the list of images that would be mirrored, as well as any images that would be pruned from the mirror registry. A dry run also allows you to catch any errors with your image set configuration early or use the generated list of images with other tools to carry out the mirroring operation.
+You can use oc-mirror to perform a dry run, without actually mirroring any images. A dry run means you can review the list of images to be mirrored. You can catch any errors with your image set configuration early by using a dry run, or use the generated list of images with other tools to conduct mirroring.
 
 .Prerequisites
 
 * You have access to the internet to obtain the necessary container images.
-* You have installed the OpenShift CLI (`oc`).
-* You have installed the oc-mirror CLI plugin.
-* You have created the image set configuration file.
+* You installed the {oc-first}.
+* You installed the oc-mirror CLI plugin.
+* You created the image set configuration file.
 
 .Procedure
 
 . Run the `oc mirror` command with the `--dry-run` flag to perform a dry run:
 +
 [source,terminal]
 ----
-$ oc mirror --config=./imageset-config.yaml \// <1>
-  docker://registry.example:5000            \// <2>
-  --dry-run                                  <3>
+$ oc mirror --v2 --authfile ~/.pull-secret.json  --dry-run -c ./isc.yaml --workspace file://./wspace docker://mirror.com
 ----
-<1> Pass in the image set configuration file that was created. This procedure assumes that it is named `imageset-config.yaml`.
-<2> Specify the mirror registry. Nothing is mirrored to this registry as long as you use the `--dry-run` flag.
-<3> Use the `--dry-run` flag to generate the dry run artifacts and not an actual image set file.
+where:
+
+`isc.yaml`:: Specifies the name of the image set configuration file that you created.
+`file://./wspace docker://mirror.com`:: Specifies the mirror registry. Nothing is mirrored to this registry when you use the `--dry-run` flag.
+`--dry-run -c`:: The dry run flag generates the dry run artifacts and not an actual image set file.
```

Comment:
--dry-run is one flag to perform dry-run and -c is to specify the imagesetconfig file, these both cannot be in a single line.